CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.0.0
+
+- Drop python 2.7 support.
```

Comment:
@bishopbm1 is this entry ok? What else do we need to say?

Reply:
Nope thats perfect! I'm good. Thanks!